app/src/main/java/org/simple/clinic/signature/SignatureActivity.kt

```diff
@@ -1,18 +1,82 @@
 package org.simple.clinic.signature
 
+import android.content.Context
+import android.graphics.Bitmap
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.jakewharton.rxbinding3.view.clicks
+import io.reactivex.rxkotlin.ofType
 import kotlinx.android.synthetic.main.activity_signature.*
+import org.simple.clinic.ClinicApp
 import org.simple.clinic.R
+import org.simple.clinic.mobius.MobiusDelegate
+import org.simple.clinic.util.unsafeLazy
+import javax.inject.Inject
 
 class SignatureActivity : AppCompatActivity(), SignatureUiActions {
+
+  private lateinit var component: SignatureComponent
+
+  @Inject
+  lateinit var effectHandlerFactory: SignatureEffectHandler.Factory
+
+  private val events by unsafeLazy {
+    savingSignature()
+  }
+
+  private val mobiusDelegate by unsafeLazy {
+
+    MobiusDelegate.forActivity(
+        events = events.ofType(),
+        defaultModel = SignatureModel.create(filesDir),
+        update = SignatureUpdate(),
+        effectHandler = effectHandlerFactory.create(this).build()
+    )
+  }
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_signature)
   }
 
+  private fun savingSignature() = acceptSignature
```

Comment:
You can do it in next PR, but we usually name it as `acceptSignatureClicks` or `undoClicks` to convey these are view click streams.